docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -348,6 +348,313 @@ and a file system location of ``/var/my_tables/test_table``::
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
 
+.. _iceberg-metadata-tables:
+
+Metadata tables
+---------------
+
+The Iceberg connector maintains several hidden tables that provide metadata
+for a specific table. You can query each metadata table by appending the
+metadata table name to the table name::
+
+   SELECT * FROM "test_table$data"
+
+``$data`` table
+^^^^^^^^^^^^^^^
+
+The ``$data`` table is an alias for the Iceberg table.
+
+The statement::
+
+    SELECT * FROM "test_table$data"
+
+is equivalent to::
+
+    SELECT * FROM test_table
+
+``$properties`` table
+^^^^^^^^^^^^^^^^^^^^^
+
+The ``$properties`` table provides access to general information about iceberg
+table configuration and any additional metadata key/value pairs that the table
+was tagged with.
+
+You can retrieve the changelog of the Iceberg table ``test_table``
+by using the query::
+
+    SELECT * FROM "test_table$properties"
+
+.. code-block:: text
+
+     key                   | value    |
+    -----------------------+----------+
+    write.format.default   | PARQUET  |
+    format-version         | 2        |
+
+``$history`` table
+^^^^^^^^^^^^^^^^^^
+
+The ``$history`` table provides a log of the metadata changes performed on
+the Iceberg table.
+
+You can retrieve the changelog of the Iceberg table ``test_table``
+by using the query::
+
+    SELECT * FROM "test_table$history"
+
+.. code-block:: text
+
+     made_current_ad    | snapshot_id          | parent_id            | is_current_ancestor
+    --------------------+----------------------+----------------------+--------------------
+    2022-01-10 08:11:20 | 8667764846443717831  |  <null>              |  true
```

Comment:
Example output suggests it is not timestamtp **with time zone**. Maybe it makes sense for brevity IDK.

Reply:
Copy-pasted the output from IntelliJ sql browser which doesn't show by default the timezone. I modified it now.